### PR TITLE
WEBAPP-256: Add flow to endpoint module

### PR DESCRIPTION
**Pull request by Schedulaar**
This PR only introduces only easy-to add types.
For better types for all the actions, reducers, etc. I think we would need to do either heavy refactorings or too many hardcoded types...
Not sure how to proceed here...

### CODE REVIEW

File: src/modules/endpoint/models/__tests__/CategoriesMapModel.spec.js [hunk 1]
```diff
@@ -21,26 +23,49 @@ describe('CategoriesMapModel', () => {
...
   const categories = [
-    new CategoryModel({id: 0, path: '/augsburg/de', title: 'augsburg', parentPath: ''}),
+    new CategoryModel({
+      id: 0,
+      path: '/augsburg/de',
+      title: 'augsburg',
+      parentPath: '',
+      availableLanguages: new Map(),
+      content: 'exampleContent0',
+      lastUpdate: moment(),
```
Comment (NKlug) [inline src/modules/endpoint/models/__tests__/CategoriesMapModel.spec.js]: Not sure if this is important but using `moment()` here makes the tests a bit non-deterministic. Maybe a fixed date and time would be better.

File: src/modules/endpoint/models/__tests__/CategoryModel.spec.js [hunk 2]
```diff
@@ -9,7 +12,8 @@ describe('CategoryModel', () => {
     parentPath: '/augsburg/en',
     order: 75,
     availableLanguages: new Map([['de', '/augsburg/de/']]),
-    thumbnail: 'https://cms.integreat-ap…/03/Hotline-150x150.png'
+    thumbnail: 'https://cms.integreat-ap…/03/Hotline-150x150.png',
+    lastUpdate: moment()
```
Comment (NKlug) [inline src/modules/endpoint/models/__tests__/CategoryModel.spec.js]: Same in this file and in some others.

File: src/routes/feedback/components/FeedbackBoxContainer.js [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+import FeedbackDropdownItem from '../FeedbackDropdownItem'
+import { WOHNEN_ROUTE } from '../../../modules/app/routes/wohnen'
+import { SPRUNGBRETT_ROUTE } from '../../../modules/app/routes/sprungbrett'
+import { EXTRAS_ROUTE } from '../../../modules/app/routes/extras'
+import ExtraModel from '../../../modules/endpoint/models/ExtraModel'
+import FeedbackBox from './FeedbackBox'
+import { translate } from 'react-i18next'
+import type { FeedbackDataType } from '../../../modules/endpoint/FeedbackEndpoint'
+
+type PropsType = {
```
Comment (NKlug) [inline src/routes/feedback/components/FeedbackBoxContainer.js]: Afaik we want to use `{|…|}` for the props, right?

Comment (klinzo) [reply src/routes/feedback/components/FeedbackBoxContainer.js]: I already did this in #201.

**Pull request merged**